seisidd/experiment.py

```diff
@@ -321,10 +425,46 @@ def tomo_scan(self, cfg):
         )
         n_projections = len(angs)
         cfg['tomo']['n_projections'] = n_projections
-        total_images  = n_white + n_projections + n_white + n_dark
+        cfg['tomo']['total_images']  = n_white + n_projections + n_white + n_dark
         fp = cfg['output']['filepath']
         fn = cfg['output']['fileprefix']
-        
+
+        # consider adding an extra step to:
+        #   Perform energy calibration, set intended attenuation
+        #   set the lenses, change the intended slit size
+        #   prime the control of FS
+
+        #############################################
+        ## step 0.1: check and set beam parameters ##
+        #############################################
+        # set slit sizes
+        # These are the 1-ID-E controls
+        #   epics_put("1ide1:Kohzu_E_upHsize.VAL", ($1), 10) ##
+        #   epics_put("1ide1:Kohzu_E_dnHsize.VAL", (($1)+0.1), 10) ##
+        #   epics_put("1ide1:Kohzu_E_upVsize.VAL", ($2), 10) ## VERT SIZE
+        #   epics_put("1ide1:Kohzu_E_dnVsize.VAL", ($2)+0.1, 10) ##
+        _beam_h_size    =   cfg['tomo']['beamsize_h']
```

Comment:
adjust the beam size during the Tomo scan?  Is there any particular good reason for doing this?

Reply:
Will remove this.